fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/PlannerRepl.java

```diff
@@ -156,7 +169,9 @@ public void onRegisterQuantifier(@Nonnull final Quantifier quantifier) {
     @Override
     public void onInstall() {
         try {
-            terminal = TerminalBuilder.builder().build();
+            if (terminal == null) {
```

Comment:
Why not always insist on passing in the Terminal? Can we invert this and make the `terminal` `NonNull`?

Reply:
Sure, I can also do that.